src/manager/app/models/volume.py

```diff
@@ -24,18 +29,23 @@ def to_container_create_kwargs(self) -> dict | None:
         if self.host_mount_prefix is not None:
             return {
```

Comment:
can't we set `self.volumes=['data','config',...]` and do this?
`return {f"{self.host_mount_path}/{volume}: {...} for volume in self.volumes}`